src/tracer/models.ts

```diff
@@ -0,0 +1,22 @@
+import { Event } from '../client';
+import { Evaluation as TestingEvaluation } from '../testing';
+import { BaseEventEvaluator } from '../testing/models';
+import { ArbitraryProperties, PromptTracking } from '../types';
+
+export type TracerEvent = Omit<Event, 'id' | 'traceId'> & { traceId?: string };
+export type Evaluation = TestingEvaluation;
```

Comment:
why are we importing like `import { Evaluation as TestingEvaluation } from '../testing';` and then renaming back to `Evaluation` here?

Reply:
I wasn't following our exporting strategy first - I will clean this up